components/demo/demo-snippet.js

```diff
@@ -82,6 +96,7 @@ class DemoSnippet extends LitElement {
 
 	render() {
 		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
+		const fullscreen = this.fullscreen ? html`<d2l-switch text="Fullscreen" ?on="${this.fullscreenView}" @change="${this._handleFullscreenViewChange}"></d2l-switch><br />` : null;
```

Comment:
Maybe we want this switch to always be present? This existing `fullscreen` property just sorta means full width (unfortunate name), and I can imagine it being useful to toggle fullscreen even when the snippet is standard width.

Reply:
Yeah, I considered this. I'll gladly make that change.

Reply:
Though, I guess we have to keep it named differently so as not to be breaking. There are only a couple uses of the existing `fullscreen`. Perhaps we can get those updated in tandem.

Reply:
Yeah, if we can easily identify them, I'm all for updating the property name.